doc/quickstart.rst

```diff
@@ -52,8 +52,37 @@ One option is to simply run ``suricata-update`` as root or with
           the ``suricata`` group then add users to the ``suricata``
           group.
 
-	  More documentation will be provided about this, including a
-	  tool to verify and maybe setup the permissions.
+Steps to setup the above directories with the correct permissions:
+
+First, create a group ``suricata``::
+
+    sudo groupadd suricata
+
+Next, change the group of the directories and its files recursively::
+
+    sudo chgrp -R suricata /etc/suricata
+    sudo chgrp -R suricata /var/lib/suricata/rules
+    sudo chgrp -R suricata /var/lib/suricata/update
+
+Setup the directories with the correct permissions for the ``suricata``
+group::
+
+    sudo chmod -R g+r /etc/suricata/
```

Comment:
Hmm most of the configuration like update.yaml, modify.yaml goes here so it should not just have read access.

Reply:
So, it should have both read and write access? Then, there should also be a change in the docs where it is written `Directory /etc/suricata: read access`. Should I consider updating it?